### PR TITLE
various fixes and features: synchronize rules from local dir, check service status after restart, rename port ansible var, add become where needed allow use of SSL elasticsearch cluster connection

### PR DESCRIPTION
**Pull request by NanoPish**
* automatic rules upload -> local dir {{ elastalert_local_rules_dir }}
* automatic rules upload: delete rules on server that are not in the {{ elastalert_local_rules_dir }} ansible machine directory (=> synchronize ~= rsync)
* automatic service status check after restart -> see handlers
* add become: yes everywhere it is needed to avoid unecessary become
* rename {{ port }} to {{ es_port }} to make it clearer
* SSL elasticsearch connection

### CODE REVIEW

**Comment by NanoPish** [comment]
Hey, are you interested in those features?

File: vars/main.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 host_name: localhost
-port: 9300
+es_port: 9300
```
Comment (iamabhishek-dubey) [inline vars/main.yml]: Let's keep it 9200 because 9300 generally is for TCP connections

**Review by iamabhishek-dubey — CHANGES_REQUESTED** [review]
Just one change

**Comment by NanoPish** [comment]
Hey! I updated the documentation with all the variables related to automatically uploading and deleting rules, using SSL...
I also added examples and version change in the README.md.
I let you tag the release

**Comment by NanoPish** [comment]
I also tested the playbook with and without rule upload to check everything is okay
tell me if its all good

**Comment by iamabhishek-dubey** [comment]
All things look good, can you resolve merge conflicts?

**Comment by NanoPish** [comment]
done

**Comment by NanoPish** [comment]
good?

**Review by iamabhishek-dubey — APPROVED** [review]

**Comment by iamabhishek-dubey** [comment]
Thanks for your contribution!!

**Pull request merged**